Brokkr/src/main/cpp/subsystems/Wrist.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "subsystems/Wrist.h"
+#include <frc/smartdashboard/SmartDashboard.h>
+
+Wrist::Wrist(WPI_TalonFX& WristMotor)
+: WristMotor(WristMotor)
+{
+    WristMotor.Config_kP(0, 0, 0);
+    WristMotor.Config_kI(0, 0, 0);
+    WristMotor.Config_kD(0, 0, 0);
+    WristMotor.Config_kF(0, 0, 0);
+}
+// This method will be called once per scheduler run
+void Wrist::Periodic() 
+{
+    frc::SmartDashboard::PutBoolean("IsWristFlicked", IsWristFlicked());
+}
+
+double Wrist::TurnWrist(double pos) 
+{
+    WristMotor.Set(ControlMode::Position, pos);
+    return pos;
+}
+
+bool Wrist::IsWristFlicked() 
+{
+    if (WristMotor.GetSelectedSensorPosition() <= 100) //placeholder
```

Comment:
Let's use constants for the limits a name.

Returning `true` and `false` is usually not needed. The result of the test can be returned. e.g.

```
bool Wrist::IsWristFlicked() 
{
    const double kWristFlickedLimit = 100.0;  // placeholder
    double sensorPosition = WristMotor.GetSelectedSensorPosition();
    return sensorPosition <= kWristFlickedLimit;
}
```